src/convert.jl

```diff
@@ -0,0 +1,228 @@
+module Conversion
+
+using ..SAM
+using ..BAM
+
+using BioSequences, BioAlignments
+
+global  const op_to_code = Dict(
```

Comment:
No need to put global here - also, usually, global constants are upper case
```suggestion
const OP_TO_CODE = Dict(
```